frontend/tests/playwright/product/product_breadcrumb.spec.ts

```diff
@@ -68,6 +68,17 @@ test.describe('Product breadcrumb test', () => {
             ).not.toBeNull();
          }
          expect(ancestorLinksCount).toBeLessThanOrEqual(3);
+
+         // If the product has less than or equal to 3 breadcrumb links, we don't display the breadcrumb menu
+         const desktopBreadcrumbMenu = page.getByTestId(
+            'breadcrumb-menu-desktop'
+         );
+         await expect(desktopBreadcrumbMenu).not.toBeVisible();
```

Comment:
If we are not using the set initialized variable later in the code why not pass the retrieval of the element directly to the assertion?

```suggestion
         await expect(page.getByTestId(
            'breadcrumb-menu-desktop'
         )).not.toBeVisible();
```

Reply:
Thanks @ardelato for the suggestion! That makes it very cleaner! Updated to use the elements directly for the unused variables in https://github.com/iFixit/react-commerce/pull/1227/commits/7e5c2b4f96ac281a72e3117878516acd67cc5d31.